objects/obj_star_select/Step_0.gml

```diff
@@ -7,24 +7,25 @@ if (instance_exists(target)){
     x=target.x;y=target.y;
     
     // Buttons that are available
-    
+    var enema=false;
     if (obj_controller.selecting_planet>0){
         if (target.present_fleet[1]=0)/* and (target.p_type[obj_controller.selecting_planet]!="Dead")*/{
-            var enema;enema=false;
             if (target.p_owner[obj_controller.selecting_planet]>5) then enema=true;
             if (obj_controller.faction_status[target.p_owner[obj_controller.selecting_planet]]="War") then enma=true;
             
             if (target.p_player[obj_controller.selecting_planet]>0){
-                if (enema=true){button1="Attack";button2="Purge";}
+                if (enema){
+                    button1="Attack";
+                    button2="Purge";
+                }
             }
         }
         if (target.present_fleet[1]>0)/* and (target.p_type[obj_controller.selecting_planet]!="Dead")*/{
-            var enema;enema=false;
             if (target.p_owner[obj_controller.selecting_planet]>5) then enema=true;
             if (obj_controller.faction_status[target.p_owner[obj_controller.selecting_planet]]="War") then enma=true;
             
-            if (enema=true){button1="Attack";button2="Raid";button3="Bombard";}
-            if (enema=false){button1="Attack";button2="Raid";button3="Purge";}
+            if (enema){button1="Attack";button2="Raid";button3="Bombard";}
+            if (!enema){button1="Attack";button2="Raid";button3="Purge";}
```

Comment:
you can just write `else`, instead of `if (!enema)`, if we know that `enema` is boolean